python/pyspark/sql/tests.py

```diff
@@ -830,12 +830,12 @@ def test_infer_long_type(self):
         self.assertEquals(100000000000000, df1.first().f2)
 
         self.assertEqual(_infer_type(1), LongType())
-        self.assertEqual(_infer_type(2**10), LongType())
-        self.assertEqual(_infer_type(2**20), LongType())
-        self.assertEqual(_infer_type(2**31 - 1), LongType())
-        self.assertEqual(_infer_type(2**31), LongType())
-        self.assertEqual(_infer_type(2**61), LongType())
-        self.assertEqual(_infer_type(2**71), LongType())
+        self.assertEqual(_infer_type(1<<10), LongType())
```

Comment:
Ah sorry, I though the earlier comment was to update the 2** to 1<< everywhere in pyspark (since there were not a lot of places.) I'll revert this change.